### ISSUE TITLE
`validatePlugin is not a function` with mocha 8.2.0

### ISSUE THREAD

**Issue by ffflorian** [jprichardson/electron-mocha#165]
When using electron-mocha v9.1.0 with mocha v8.2.0 installed, I get the following error message:

```
$ electron-mocha --require .babel-register.js "electron/src/**/*.test?(.main).ts" --no-sandbox

✖ ERROR: validatePlugin is not a function
error Command failed with exit code 1.
```

**Comment by marc2332** [comment]
This happens because you are using mocha v8.2.0 which has renamed validatePlugin to validateLgacyPlugin. 
Code: https://github.com/mochajs/mocha/blob/f23035d070229e8f45d6fec2ff35e82b9ed7dcbc/lib/cli/run-helpers.js#L204

Anyway, electron-mocha is not intended to work alongside mocha since it already bundles it's own version, so there is no point on installing mocha manually, try removing it, `npm r mocha`, and see if it works. 👍🏻

**Comment by ffflorian** [comment]
Hi @marc2332,

> try removing it, npm r mocha, and see if it works. 👍🏻

I need the mocha standalone for another test so I can't just remove it. I'll just wait for @jprichardson to update the bundled mocha version :)

**Comment by inukshuk** [comment]
Thanks for reporting -- should be fixed in 9.3.0.

**Comment by ffflorian** [comment]
Thank you!
